core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -472,12 +512,12 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
         }}
         {if (hasInput) {
           <td sorttable_customkey={inputSortable}>
-            {inputReadable}
+            {s"$inputReadable / $inputRecords"}
```

Comment:
from the way the json serialization is done, I think you'll get something weird here when loading data from old runs that didn't have this field.  Each task is assigned a `-1`, so you end up with some negative number at the stage summary.  eg., I just ran a little test and I see "Input Size / Records" as "160.0 B / -8".